### ISSUE TITLE
bug: Ion-infinite-scroll doesn't work with ionic 5.0.0

### ISSUE THREAD

**Issue by pairmix** [ionic-team/ionic#19100]
# Bug Report

**Ionic version:**
[x] 5.0.0

**Current behavior:**
Scrolling doesn't work, only the first "page" is displayed: 25 elements

**Expected behavior:**
Scrolling must continue until 1000 elements been diplayed

```
import { Component, ViewChild } from '@angular/core';
import { IonInfiniteScroll } from '@ionic/angular';
 
@Component({
  selector: 'app-home',
  templateUrl: 'home.page.html',
  styleUrls: ['home.page.scss'],
})
export class HomePage {
 
  @ViewChild(IonInfiniteScroll, {static: false}) infiniteScroll: IonInfiniteScroll;
 
 
  dataList:any;
 
  constructor() {
    this.dataList = [];
    
    for (let i = 0; i < 25; i++) { 
      this.dataList.push("Item number "+this.dataList.length);
    }
  }
 
  loadData(event) {
    
    setTimeout(() => {
      console.log('Done');
      for (let i = 0; i < 25; i++) { 
        this.dataList.push("Item number "+this.dataList.length);
      }
      event.target.complete();
 
      // App logic to determine if all data is loaded
      // and disable the infinite scroll
      if (this.dataList.length == 1000) {
        event.target.disabled = true;
      }
    }, 500);
  }
 
  toggleInfiniteScroll() {
    this.infiniteScroll.disabled = !this.infiniteScroll.disabled;
  }
}
```

````
<ion-header>
  <ion-toolbar>
    <ion-title>
      Ionic Blank
    </ion-title>
  </ion-toolbar>
</ion-header>
 
<ion-content padding>
  <ion-button (click)="toggleInfiniteScroll()" expand="block">
    Toggle Infinite Scroll
  </ion-button>
 
  <ion-list>
    <ion-item *ngFor="let item of dataList">
      <ion-label>{{item}}</ion-label>
    </ion-item>
  </ion-list>
 
  <ion-infinite-scroll threshold="100px" (ionInfinite)="loadData($event)">
    <ion-infinite-scroll-content
      loadingSpinner="bubbles"
      loadingText="Loading more data...">
    </ion-infinite-scroll-content>
  </ion-infinite-scroll>
</ion-content>
````

$ionic info:
```
Ionic:

   Ionic CLI                     : 5.0.0 (/usr/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.6.0
   @angular-devkit/build-angular : 0.13.9
   @angular-devkit/schematics    : 7.3.9
   @angular/cli                  : 7.3.9
   @ionic/angular-toolkit        : 1.5.1

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   NodeJS : v9.11.2 (/usr/bin/node)
   npm    : 5.6.0
   OS     : Linux 4.15

```

**Comment by arjunyel** [comment]
If you're using Angular I believe you should be using the CDK infinite scroll instead https://material.angular.io/cdk/scrolling/overview

**Comment by pairmix** [comment]
Does it mean that Ion-infinite-scroll doesn't work properly ?

**Comment by FernetB** [comment]
Ionic framework current version is 4.7.4. the version that you are saying is the CLI version that has nothing to do with ion infinite scroll.

The bug you described was fixed in the ionic 4.6.1 version. (bug with the event listener) and you have the 4.6 version.

Please update the framework and see if the problem disappears.

PD: i recommend you to check the release notes once in a  while --https://ionicframework.com/docs/release-notes 

Cheers!

**Comment by pairmix** [comment]
Works fine now!
Thank you very much !

````
$ionic info                                                     

Ionic:

   Ionic CLI                     : 5.0.0 (/usr/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.7.4
   @angular-devkit/build-angular : 0.802.2
   @angular-devkit/schematics    : 8.2.2
   @angular/cli                  : 8.2.2
   @ionic/angular-toolkit        : 2.0.0

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   NodeJS : v12.8.1 (/usr/local/bin/node)
   npm    : 6.10.3
   OS     : Linux 4.15

````
